text/0021-versioning-releases.md

```diff
@@ -0,0 +1,45 @@
+# Releasing and Versioning of Openmrs SingleSpa Apps
+- Start Date: 2020/01/21
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/1
+
+## Definition
+OpenMRS: refers to the openmrs community.
+
+Micro-Frontends Squad: refers to the OpenMRS Community members responsible for creating and maintaining OpenMRS SPAs, under the OpenMRS Micro-Frontends Project.
+
+OpenMRS SPA: refers to an OpenMRS Micro-Frontend app. An example is openmrs-esm-patient-chart. See https://github.com/openmrs/openmrs-esm-patient-chart
+
+CDN: Content Distribution Network such as jsdelivr. See https://www.jsdelivr.com/
+
+MAJOR Version: version when you make incompatible API changes
```

Comment:
Since most MFEs don't have an API, I'm going to suggest

```suggestion
MAJOR Version: There are incompatible API or configuration API changes, or major UI changes that are enabled by default
```